packages/account-v2/src/modules/IDVForm/IDVForm.tsx

```diff
@@ -36,9 +36,9 @@ export const IDVForm = ({ allowDefaultValue, allowIDVSkip, selectedCountry }: TI
 
     const { isMobile } = useDevice();
 
-    const { documents_supported } = selectedCountry;
+    const { documents_supported: documentSupported } = selectedCountry;
```

Comment:
```suggestion
    const { documents_supported: supportedDocuments } = selectedCountry;
```

as these can be more than one and `documentSupported` sounds like a boolean flag